### PR TITLE
Add +CROSS

### PR DESCRIPTION
**Pull request by idelvall**
Fixes https://github.com/earthly/lib/issues/42

### CODE REVIEW

**Review by mikejholly — APPROVED** [review]

File: rust/Earthfile [hunk 1]
```diff
@@ -109,7 +109,38 @@ COPY_OUTPUT:
...
+#   DO rust+SET_CACHE_MOUNTS_ENV
+#   DO rust+CROSS --target aarch64-unknown-linux-gnu
+CROSS:
+    FUNCTION
+    ARG --required target
+    ARG args="build --release"
+    ARG output="$target/release/[^\./]+"
+    ARG version=0.2.5
+    DO +SET_CACHE_MOUNTS_ENV --target_cache_suffix="$target"
+    RUN echo "$EARTHLY_RUST_TARGET_CACHE"
```
Comment (alexcb) [inline rust/Earthfile:130]: is this intentional?

Comment (idelvall) [reply rust/Earthfile:130]: nop, thanks!

Comment (alexcb) [reply rust/Earthfile:130]: maybe it's a dumb question, but should there also be an `ARG EARTHLY_RUST_TARGET_CACHE` somewhere? I can't seem to find one while grepping.

Comment (alexcb) [reply rust/Earthfile:130]: nevermind, it's an ENV:

```
rust/Earthfile:    ENV EARTHLY_RUST_TARGET_CACHE="type=cache,mode=0777,id=$EARTHLY_CACHE_PREFIX#target#$EARTHLY_TARGET_NAME#$TARGETPLATFORM#$target_cache_suffix,sharing=locked,target=target"
```

:)

**Pull request merged**